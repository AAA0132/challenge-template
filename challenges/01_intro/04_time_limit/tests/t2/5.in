497155741629580644253
