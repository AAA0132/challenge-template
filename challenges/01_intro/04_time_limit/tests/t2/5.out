123581915717629172411292078152826061286131
